lib/fathead/perl_doc/Parsers/parse-faq.pl

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env perl
```

Comment:
I've been trying to gather all this into parse.pl rather than have a bunch of scripts all doing almost the same thing.